index.php

```diff
@@ -1,20 +1,20 @@
 <?php get_header(); ?>
 
-	<div class='main-section'>
-	  <?php get_sidebar(); ?>
-	  <ul class="posts">
-		<?php 
-		  if ( have_posts() ) :
-	      	while ( have_posts() ) : the_post();
-		?>      
-	    <li>
-		  <?php the_content('&raquo;'); ?>
-		</li>
-		<?php 
-			endwhile;
-			endif;
-		?>
-	  </ul>
-	</div>
+  <div class='main-section'>
+    <?php get_sidebar(); ?>
+    <ul class="posts">
+      <?php 
+        if ( have_posts() ) :
```

Comment:
Quick philosophical question: what would your markup look like in the off chance there are 0 posts?